kwc-picker.js

```diff
@@ -221,6 +242,9 @@ class KwcPicker extends PolymerElement {
             };
         }
     }
+    _computeCheck(key, index) {
```

Comment:
I didn't get right away that it was to hide the check mark. Maybe a name like `isCheckHidden` would make more sense